src/__tests__/Header.test.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import Header from '../components/Header';
+
+describe('Testing the Header component:', () => {
+  test('should renerd correctly', () => {
+    render(<Header />, { wrapper: BrowserRouter });
+    expect(screen).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] You need to either remove or add passing tests please
![image](https://user-images.githubusercontent.com/62705947/199999186-68f143c2-dca7-44c8-a509-1fa83f7f9396.png)

Reply:
Hi @GzimAsani 👋

Thank you for mentioning this, I was wondering what should I do with it he whole time 😅.
I will do so 😎